misc/triage/labels.yaml

```diff
@@ -35,6 +35,9 @@ labels:
 - name: kind/security
   color: f4dddc
   description: security issues
+- name: vulnerability
+  color: e11d21
+  description: Categorizes issue or PR as related to vulnerabilities.
```

Comment:
I want to put it under `others`.
https://github.com/aquasecurity/trivy/blob/22e562e55c34a0727c59860e099316ab6911c00a/misc/triage/labels.yaml#L139-L145

Also, we should emphasize it is related to Trivy's vulnerabilities.

```suggestion
  description: Categorizes issue or PR as related to Trivy's own vulnerabilities.
```

Reply:
Done